### ISSUE TITLE
ImportError: cannot import name 'unpacker_dir'

### ISSUE THREAD

**Issue by guange2015** [justbilt/untp#21]
untp
Traceback (most recent call last):
  File "d:\soft\miniconda3\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "d:\soft\miniconda3\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "D:\soft\miniconda3\Scripts\untp.exe\__main__.py", line 4, in <module>
  File "d:\soft\miniconda3\lib\site-packages\untp\__init__.py", line 3, in <module>
    from untp import unpacker_dir, unpacker
ImportError: cannot import name 'unpacker_dir' from 'untp' (d:\soft\miniconda3\lib\site-packages\untp\__init__.py)

**Comment by justbilt** [comment]
please check your untp version via `pip show untp`, i tested on 1.1.7 and it work fine. this is my test code:

```
from untp import unpacker_dir, unpacker

print unpacker_dir
print unpacker
```

and  output:

```
python test.py
<function unpacker_dir at 0x1038611b8>
<function unpacker at 0x103861140>
```

---

you could you use `pip install untp --upgrade` to upgrade to lasest version.

**Comment by justbilt** [comment]
the issue fixed and untp 1.1.8 released, you could got it via `pip install untp --upgrade`.
